examples/skills_tuning/instructlab/structured_summary.py

```
@@ -0,0 +1,216 @@
+# ---
+
+# %%
+# %load_ext autoreload
+# %autoreload 2
+
+# %%
+# Standard
+import random
+
+# Third Party
+from datasets import Dataset, load_dataset
```

Flake8 (7.2.0)

[error] 'datasets.Dataset' imported but unused

(F401)